src/ServiceControl.Monitoring/Licensing/LicenseManager.cs

```diff
@@ -14,10 +14,10 @@ public class LicenseManager
         public void Refresh()
         {
             Logger.Debug("Checking License Status");
-            var result = ActiveLicense.Find("ServiceControl",
-                new LicenseSourceHKLMRegKey(),
```

Comment:
Is HKLM already covered?

Reply:
@mikeminutillo Yes it is. As long as the `REGISTRYLICENSESOURCE` symbol is defined, then the call to `LicenseSource.GetStandardLicenseSources()` includes standard file locations as well as HKLM and HKCU locations.

Reply:
We had to add the custom code within the SCMU because we were _writing_ the license, not just reading it.